error_test.go

```diff
@@ -32,6 +33,7 @@ func TestErrDecimal(t *testing.T) {
 	ed.Round(a, a)
 }
 
+// REVIEW: this is testing something more specific than "NewErrDecimal", right?
```

Comment:
it's really a test of c.WithPrecision generating a copy. Added a comment and renamed.